packages/driver/src/cypress/events.ts

```diff
@@ -0,0 +1,119 @@
+import _ from 'lodash'
+import { EventEmitter2 } from 'eventemitter2'
+import Debug from 'debug'
+import Bluebird from 'bluebird'
```

Comment:
You might want to use a * here since bluebird is `export = `
```suggestion
import * as Bluebird from 'bluebird'
```